packages/pharos/src/components/toast/pharos-toaster.ts

```diff
@@ -79,12 +88,22 @@ export class PharosToaster extends PharosElement {
     }
   }
 
+  private _focusOnReturnElements(returnElements: Array<HTMLElement>): void {
+    for (const element of returnElements) {
+      if (element && element.isConnected) {
```

Comment:
can't remember if pharos uses it too much but you could use optional chaining here

```suggestion
      if (element?.isConnected) {
```